### PR TITLE
Update GOVERNANCE.md

### PR DESCRIPTION
**Pull request by Frenchhorn005**
Added David Mongeau as a new member to the TSC

Signed-off-by: Andre de Waal <andre.dewaal@ibm.com>

### CODE REVIEW

**Comment by Frenchhorn005** [comment]
Definitely. I will add him.

File: COMMITTERS.csv [hunk 1]
```diff
@@ -3,3 +3,4 @@ John Mertic,jmertic@linuxfoundation.org,jmertic
 James Harroun,james.harroun@sas.com,ProcJimi
 Andre de Waal,andre.dewaal@ibm.com,Frenchhorn005
 Goutam Chakraborty,goutam.chakraborty@okstate.edu,GoutamChakraborty
+David Mongeau,mongeau@berkeley.edu
```
Comment (jmertic) [inline COMMITTERS.csv]: does he have a GitHub ID as well?

**Comment by Frenchhorn005** [comment]
John,

Not yet. I will write to him and ask to send me his GitHub ID.

Thanks,

Andre de Waal
Ecosystem Data Science and AI Learning University Programs Lead
IBM Cloud and  Cognitive Software
Durham, NC 27703, USA
Phone: +1 (919) 633-1424
E-mail: andre.dewaal@ibm.com

**Comment by Frenchhorn005** [comment]
I will write to David to ask for his GitHub ID.

**Comment by jmertic** [comment]
Thanks! Once you get that added let me know and I'll get it synced up.

**Comment by jmertic** [comment]
@GoutamChakraborty and @ProcJimi Can you add your approval and I can then merge.

**Comment by GoutamChakraborty** [comment]
Approved

Dr. Goutam Chakraborty
SAS Professor of Marketing Analytics
Director, MS in Business Analytics
Oklahoma State University
Phone: 405 744 7644
Web: analytics.okstate.edu

Sent from my phone. please excuse brevity and typos.

________________________________

**Pull request merged**